docs: clarify path to database password reset in dashboard (#45853)
supabase/supabase: apps/docs/content/troubleshooting/how-do-i-reset-my-supabase-database-password-oTs5sB.mdx

database_id = "6e4a515c-5e70-4b92-808c-f076d47d4e35"
---

> You can reset your database password by navigating to [Database > Settings](/dashboard/project/_/database/settings) from the project dashboard.

Reviewer: ```suggestion
You can reset your database password from the [**Database > Settings**](/dashboard/project/_/database/settings) section of the Dashboard.
```

A completely undocumented consistency change we are trying to make.

Author: That is fair. I notice this is the second instance of feedback recently saying it's not accurate but I've confirmed the original guidance is accurate. Perhaps I should cancel this PR.

Reviewer: @tk1ng Oh no, this is me making a formatting change, nothing to do with anything else. The change you made was also fine, I am just tweaking for consistency.

Something like **Section > Section** page/section of the Dashboard. I have a theory that by making it explicit will also mean it's better for agents, etc

Author: Ah, got it. I misunderstood. Nevermind me lol
